.env

```
@@ -1 +1,5 @@
+API=https://exam.elevateegy.com/api/v1
 NEXT_PUBLIC_API_URL=https://exam.elevateegy.com/api/v1
+NEXT_PUBLIC_Basic_API_URL="http://localhost:3000/api"
+NEXTAUTH_URL="http://localhost:3000"
+NEXTAUTH_SECRET="5ZRcQfeGv3kkmWsf23P/1XkkDS2xsiVQlZdJiH/Vz9M="
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)